gap/TGCellModelGraph.gd

```diff
@@ -194,6 +196,21 @@ DeclareGlobalFunction( "AddOrientedNNNEdgesToTessellationModelGraph" );
 #! (see <Ref Sect='Section_TGCellModelGraph'/>).
 DeclareGlobalFunction( "KagomeModelGraph" );
 
+#! @Description
+#!   Constructs the ${p,q}$-Lieb model graph derived from the `TGCellGraph`
+#!   (see <Ref Sect='Section_TGCellGraph'/>) <A>cellgraph</A> of the triangle group
+#!   $(2,q,p)$ associated with <A>cellgraph</A>.
```

Comment:
Do I understand correctly that Lieb lattices can only be constructed for $p\geq q$, i.e., no lattices from the dual tessellation? If yes, it might be worth commenting on that here and/or in other places.

Reply:
While I'm implicitly assuming that in $\Delta(2,q,p)$, $p\geq q$, this is a bit intransparent.

Reply:
With issue #4 fixed by PR #8, it should now not be a problem to extend the implementation to the dual.

Reply:
Thank you! I will make corresponding adjustments.